pkg/controller/registry/resolver/cache/cache.go

```diff
@@ -145,6 +145,29 @@ func (c *NamespacedOperatorCache) Error() error {
 	return errors.NewAggregate(errs)
 }
 
+func (c *NamespacedOperatorCache) GetSnapshotError(namespace string, name string) error {
+	sourceKey := SourceKey{
+		Name:      name,
+		Namespace: namespace,
+	}
+
+	snapshot, ok := c.snapshots[sourceKey]
+
+	// Should this return an error?
+	// Or will the resolution fail because the catsrc hasn't been indexed yet
+	// and not bundle found?
```

Comment:
Maybe for packages references directly by a Subscription, but not necessarily for an operator that is installed to satisfy a dependency.